src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -1387,15 +1350,15 @@ namespace ngraph
                         }
                     }
```

Comment:
Not necessary, but convenient: could you delete lines 1344 - 1351, which are no longer used?


```
                     int num_nontrivial_dims = 0;
                     for (int64_t i = input_shape.size() - 1; i > 1; i--)
                     {
                         if (input_shape[i] > 1)
                         {
                             num_nontrivial_dims++;
                         }
                     }
```